Toolset/palettes/script editor/behaviors/revsecommoneditorbehavior.livecodescript

```diff
@@ -75,7 +75,7 @@ command setObjectID pObjectID
       if tRuggedID is not sObjectID then
          put tRuggedID into sObjectID
          if exists(sObjectId) and \
-               there is a stack "com.livecode.script-library.autocomplete" then
+               revEnvironmentEditionProperty("autocomplete") then
```

Comment:
This adds a dependency for use of the common behavior outside the IDE (such as in LCFM)